### PR TITLE
Rules for ads found on parianostypos.gr

### PR DESCRIPTION
**Pull request by pappasadrian**
Blocking two ad cdn's, plus a couple of cosmetic filters for randomly generated id names.

### CODE REVIEW

File: void-gr-filters.txt [hunk 1]
```diff
@@ -152,6 +152,7 @@ zougla.gr/uploads/thomas/travel_city_banner*
 ||ads.naftemporiki.gr
 ||adwebone.com^$third-party
 ||airfasttickets.gr/affiliate/*
+||aj2234.online^
```
Comment (kargig) [inline void-gr-filters.txt:155]: the relevant URL:
`https://cdn77.aj2234.online/files2234/3/3/22/t/290/gif_fallback_el-GR_300x250_novibet-greek-roulette-novibet-all-bets-blackjack.gif`
is blocked by easylist due to the `_300x250_` included there.

File: void-gr-filters.txt [hunk 2]
```diff
@@ -177,6 +178,7 @@ zougla.gr/uploads/thomas/travel_city_banner*
 ||cdn1.bbend.net/media/com_news/ads/*
 ||cdnprojectagora.s3.amazonaws.com/gr/gazzetta.js$script,domain=gazzetta.gr
 ||cheapis.gr/widget/*
+||clevernt.com^
```
Comment (kargig) [inline void-gr-filters.txt:181]: this is included in other blocklists such as Peter Lowe's adservers. Check it here: https://pgl.yoyo.org/adservers/

No need to include it in the Greek adblock list, as it's not only related to Greek sites.

**Review by kargig — CHANGES_REQUESTED** [review]
the 2 CDN rules don't belong in the Greek adblock list imo.
@aldi what do you think ?

File: void-gr-filters.txt (hunk 2 above)
Comment (pappasadrian) [reply void-gr-filters.txt:181]: Fair enough, it was included in a non-standard list that I didn't know of, thanks for bringing it to my attention

File: void-gr-filters.txt (hunk 1 above)
Comment (pappasadrian) [reply void-gr-filters.txt:155]: I have easylist enabled but it doesn't get blocked for me. IDK why not.
I'll see if I can commit the url ban there.

**Comment by pappasadrian** [comment]
In any case, yes, the two cdn filters are probably not proper for the greek adblock list. sorry for the mishap

**Review by kargig — APPROVED** [review]
LGTM

**Pull request merged**